worlds/yugioh06/docs/en_Yu-Gi-Oh! 2006.md

```diff
@@ -0,0 +1,53 @@
+# Yu-Gi-Oh! Ultimate Masters: World Championship Tournament 2006
+
+## Where is the settings page?
+
+The [player settings page for this game](../player-settings) contains all the options you need to configure and
```

Comment:
looks like it redirects fine, but should be updated to be player-options instead of player-settings in the link, and use 'options' instead of 'settings' in the docs throughout